docs/app/pages/components/components-sections/scrollbar/virtual-for/snippets/app.ts

```diff
@@ -0,0 +1,32 @@
+import { Component } from '@angular/core';
+import 'chance';
+
+@Component({
+    selector: 'app',
+    templateUrl: './app.component.html',
+    styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+
+    employees: Employee[] = [];
+
+    constructor() {
+        for (let idx = 1; idx <= 10000; idx++) {
+            this.employees.push({
```

Comment:
We discussed async loading. Could you replace this with a setTimeout-type example similar to the other virtual scroll section?